server/config/routes.js

```diff
@@ -200,6 +200,7 @@ exports.configure = function configure(app) {
   // trial balance routes
   app.post('/journal/trialbalance', trialbalance.postTrialBalance);
   app.post('/journal/togeneralledger', trialbalance.postToGeneralLedger); // TODO : rename?
+  app.put('/journal/:uuid/reverse', journal.reverse);
```

Comment:
This would be better as a POST method, since it is creating new records in the database, instead of modifying existing ones.